src/accommodation/dto/get-user-accommodations.dto.ts

```diff
@@ -0,0 +1,10 @@
+import { Transform } from 'class-transformer';
+import { IsBoolean, IsOptional } from 'class-validator';
+import { PaginationDto } from './pagination.dto';
+
+export class GetUserAccommodationsDto extends PaginationDto {
+  @Transform(({ value }) => value === 'true')
```

Comment:
check if possible to use ParseBoolPipe in controller instead, please

Reply:
i think not, cause values from query params are strings, and even if i send "true" if fails, with the message it should be boolean, any way i need to transform it in DTO. 
Or i need to create custom pipe, with checks for the values.
You suggestions?